Docs/Configuration.tex

```diff
@@ -1254,6 +1254,13 @@ \subsection{Introduction}\label{booterintro}
 which is no longer maintained. Refer to the \hyperref[troubleshootingtricks]{Tips and Tricks}
 section for instructions on migration.
 
+Booter changes apply with the following effective order:
+\begin{itemize}
+\tightlist
+\item \texttt{MmioWhitelist} and \texttt{Quirks} are processed.
```

Comment:
MmioWhitelist is part of the quirk DevirtualiseMmio

Reply:
https://github.com/acidanthera/OpenCorePkg/commit/4505bebeb58a889f788613f8673977694109086b